Actualización del ReadMe.md

Emanlui opened the pull request:
También se cambió la ruta estática por la carpeta de resources.

faoalvarado5 committed:
feature/#3_archivo_prueba2.txt

Se añadió un segundo ejemplo de prueba llamado"prueba2.txt"

Commit:
Actualización del ReadMe.md

También se cambió la ruta estática por la carpeta de resources.

Commit:
Actualización del ReadMe.md

También se cambió la ruta estática por la carpeta de resources.

natefedez commented on README.md:
```diff
@@ -1,142 +1,34 @@
-https://www.jflex.de/
-Otros enlaces importantes: Especificación del Lenguaje JAVA Completa:
-https://docs.oracle.com/javase/specs/jls/se13/html/jls-3.html#jls-3.8
+  - Emanuelle Jiménez (Emanlui)
```
Dos apellidos y quitar el nickname :P

natefedez commented on src/main/Main.py:
```diff
@@ -1,19 +1,24 @@
 
-#Abre el archivo del código que se va a procesar.
-archivo = open(ruta + sys.argv[1],'r')
+path = os.getcwd().split("\\")
```
Agregar la documentación interna adecuada para este código nuevo

natefedez commented on src/main/Main.py:
```diff
@@ -1,19 +1,24 @@
+path = os.getcwd().split("\\")
+new_path = ""
+
+for i in range(len(path)-1):
```
Se hicieron las pruebas correspondientes que es funcional en Windows y Linux?

natefedez commented on README.md:
```diff
@@ -1,142 +1,34 @@
-9. No se requiere para esta entregar implementar técnicas de corrección de errores, es decir, ante un error, el programa desplegará lo contenido en el punto 8 y finalizará la ejecución del programa.
-10. No se podrá hacer uso de analizadores léxicos automáticos o bibliotecas de análisis léxico, todo se deberá programar. Puede hacer uso de bibliotecas para el manejo de expresiones regulares con libertad.
-11. Se recomienda poner énfasis para la implementación de este proyecto los conceptos de: buffers para la lectura de archivos, tabla de identificadores.
+   - A la hora de utilizar jetbrains para programar debemos buscar la opción de terminal, esta es facilitada por el mismo idle.
```
Agregar versión de PyCharm utilizada, enlace de descarga y agregar combinación de teclas para editar configuración de corrida o hacerlo de forma manual

Emanlui replied:
Yo no tuve que editar mediante configuración de teclas

natefedez commented on README.md:
```diff
@@ -1,142 +1,34 @@
-9. No se requiere para esta entregar implementar técnicas de corrección de errores, es decir, ante un error, el programa desplegará lo contenido en el punto 8 y finalizará la ejecución del programa.
-10. No se podrá hacer uso de analizadores léxicos automáticos o bibliotecas de análisis léxico, todo se deberá programar. Puede hacer uso de bibliotecas para el manejo de expresiones regulares con libertad.
-11. Se recomienda poner énfasis para la implementación de este proyecto los conceptos de: buffers para la lectura de archivos, tabla de identificadores.
+   - A la hora de utilizar jetbrains para programar debemos buscar la opción de terminal, esta es facilitada por el mismo idle.
```
Cambiar jetbrains -> PyCharm Jetbrains [versión]

natefedez commented on README.md:
```diff
@@ -1,142 +1,34 @@
+   - El lenguaje de programación utilizado para crear dicho compilador fue python, es por eso que es necesario instalarlo en el sistema para correr el programa. El instalador se encuentra en el siguiente enlace: https://www.python.org/downloads/
 
-* **Especificación del Proyecto**: El proyecto consistirá en implementar un analizador léxico sobre un subconjunto del lenguaje Java pero en idioma español, es decir, se intentará generar un lenguaje java tropicalizado.
+   - En caso de que se utilice linux, entonces se debe escribir el siguiente comando en la terminal:
```
`linux` empieza en mayúscula

natefedez commented on README.md:
```diff
@@ -1,142 +1,34 @@
-regulares y autómatas) y su aplicabilidad para la implementación de lenguajes de
-programación.
-4. Diseñar un Analizador Léxico que procese el lenguaje a crear.
+   - El lenguaje de programación utilizado para crear dicho compilador fue python, es por eso que es necesario instalarlo en el sistema para correr el programa. El instalador se encuentra en el siguiente enlace: https://www.python.org/downloads/
```
* `python` empieza en mayúscula, bien hecho al agregar el enlace para instalar `Python`
* Agregar versión de `Python` utilizada

natefedez commented on README.md:
```diff
@@ -1,142 +1,34 @@
+[![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
 
-* **Objetivo General**: Aplicar principios, modelos y técnicas para el diseño y la construcción de procesadores de lenguajes de programación de alto nivel, con énfasis en compiladores e intérpretes, para este proyecto, enfocándose en el diseño de un Analizador Léxico.
+Este proyecto tiene como finalidad la creación de un analizador léxico de java tropicalizado al español
```
cambiar a `Java Tropicalizado al Español`

natefedez commented on README.md:
```diff
@@ -1,142 +1,34 @@
+# Cómo correr el programa
 
-* **Requerimientos de Diseño de Software**
+  - Una vez instalado python3 debemos abrir la terminal de python y correr el siguiente comando
```
`python` va en mayúscula y especificar siempre la versión de lenguaje utilizado

natefedez commented on README.md:
```diff
@@ -1,142 +1,34 @@
-###### Fecha de Entrega: 02/06/2020
-###### Proyecto #1:
-###### Creación de un Analizador Léxico – Java Tropicalizado al Español
+# Analizador Léxico
```
Escribir el nombre del Proyecto como título

faoalvarado5 committed:
Correccion

Se agrego un archivo con el resultado de la ejecucion de la prueba agregada anteriormente.

natefedez committed:
Merge pull request #6 from natanfdecastro/feature/#3_archivo_prueba2.txt

feature/#3_archivo_prueba2.txt

Commit:
Se arreglaron los cambios correspondientes

Commit:
Se arreglaron los cambios correspondientes

Commit:
Merge remote-tracking branch 'origin/feature/#2_archivo_README.md' into feature/#2_archivo_README.md
